sdk/js/src/index.ts

```diff
@@ -265,12 +265,25 @@ export class Client {
   /**
    * Fetches opportunities
    * @param chainId Chain id to fetch opportunities for. e.g: sepolia
+   * @param fromTime Minimum Unix timestamp of the returned opportunities, in microseconds
+   * @param limit Number of opportunities to return
    * @returns List of opportunities
    */
-  async getOpportunities(chainId?: string): Promise<Opportunity[]> {
+  async getOpportunities(
+    chainId?: string,
+    fromTime?: number,
+    limit?: number
+  ): Promise<Opportunity[]> {
     const client = createClient<paths>(this.clientOptions);
+    const fromTimeStr = fromTime
+      ? `${new Date(fromTime / 1000).toISOString().replace("Z", "")}${(
```

Comment:
you don't need to convert to microsecond format. I suggest getting the fromTime param as a Date type similar to what you did in the python sdk

Reply:
that's also an option. i did it this way to make it easier for the user to specify a microsecond time and have it query that, since `Date()` does not keep the microsecond-level granularity

Reply:
I don't think anybody is going to use such precise time formats anyway. If it's easier for user to use a Date object and do some simple day arithmetics with it, I would prefer accepting a Date object here.